download-data.py

```diff
@@ -382,6 +390,17 @@ def main() -> None:
             "Using both `--projects` and `--samples` options together is not supported.",
             file=sys.stderr,
         )
+        validation_error = True
+
+    if args.metadata_only and args.samples:
+        print(
+            "Using both `--metadata-only` and `--samples` options together is not supported.",
```

Comment:
I think either "both" or "together", but not (sorry) both?

```suggestion
            "Using `--metadata-only` and `--samples` options together is not supported.",
```